pkgs/by-name/ot/otb/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  cmake,
+  fetchFromGitHub,
+  makeWrapper,
+  lib,
+  stdenv,
+  swig,
+  which,
+  boost,
+  curl,
+  gdal,
+  itk_4_13,
+  libsvm,
+  libgeotiff,
+  muparser,
+  muparserx,
+  opencv,
+  perl,
+  python3,
+  shark,
+  tinyxml,
+}:
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "otb";
+    version = "9.0.0";
+
+    src = fetchFromGitHub {
+      owner = "orfeotoolbox";
+      repo = "otb";
+      rev = "${finalAttrs.version}";
+      hash = "sha256-Ut2aimQL6Reg62iceoaM7/nRuEV8PBWtOK7KFHKp0ws=";
+    };
+
+    nativeBuildInputs = [
+      cmake
+      makeWrapper
+      python3.pkgs.wrapPython
+      swig
+      which
+    ];
+
+    buildInputs = [
+      boost
+      curl
+      gdal
+      itk_4_13
+      libsvm
+      libgeotiff
+      muparser
+      muparserx
+      opencv
+      perl
+      python3
+      python3.pkgs.numpy
+      shark
+      tinyxml
+    ];
+
+
+    # https://www.orfeo-toolbox.org/CookBook/CompilingOTBFromSource.html#native-build-with-system-dependencies
+    # activates all modules and python by default
+    cmakeFlags = [
+      "-DOTB_WRAP_PYTHON=ON"
+      "-DOTB_BUILD_FeaturesExtraction=ON"
+      "-DOTB_BUILD_Hyperspectral=ON"
+      "-DOTB_BUILD_Learning=ON"
+      "-DOTB_BUILD_Miscellaneous=ON"
+      "-DOTB_BUILD_RemoteModules=ON"
+      "-DOTB_BUILD_SAR=ON"
+      "-DOTB_BUILD_Segmentation=ON"
+      "-DOTB_BUILD_StereoProcessing=ON"
+      "-DBUILD_TESTING=OFF"
+    ];
+
+
+    propagatedBuildInputs = [
+      boost
+      curl
+      gdal
+      itk_4_13
+      libgeotiff
+      libsvm
+      muparser
+      muparserx
+      opencv
+      perl
+      python3
+      python3.pkgs.numpy
+      shark
+      swig
+      tinyxml
+    ];
+
+    pythonPath = [python3.pkgs.numpy];
+
+    # wrap the otbcli with the environment variable
+    postInstall = ''
+      wrapProgram $out/bin/otbcli \
+          --set OTB_INSTALL_DIR "$out" \
+          --set OTB_APPLICATION_PATH "$out/lib/otb/applications"
+    '';
+
+    meta = {
+      description = "Orfeo ToolBox";
+      homepage = "https://www.orfeo-toolbox.org/";
+      license = lib.licenses.asl20;
+      maintainers = with lib.maintainers; lib.teams.geospatial.members ++ [daspk04];
```

Comment:
After a simple check, I found that [you wasn't a member of geospatial teams](https://github.com/NixOS/nixpkgs/blob/5a9be42754cee0d35d893cbed08737486e5f5e6d/maintainers/team-list.nix#L317) neither otb [wasn't a kind of project belong to the teams ](https://github.com/orgs/NixOS/projects/47/views/1).
Just dont add it if they doesn't claim it.

Reply:
Thank @jackyliu16 !

As this is a geospatial package (c.f)[https://github.com/imincik/geospatial-nix/issues/503#issue-2393462390], so I want sure how to add it, basically I just followed structure of [gdal](https://github.com/NixOS/nixpkgs/blob/194846768975b7ad2c4988bdb82572c00222c0d7/pkgs/development/libraries/gdal/default.nix#L292). So I will remove it.